test/e2e_node/g8r.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
```

Comment:
nit: Can we rename this script to `gubernator.sh`? I don't think g8r is a common enough prefix that people will know what it does.